app/templates/records/macros/explore_discover.html

```diff
@@ -0,0 +1,118 @@
+{% macro explore_discover(record) %}
+<div class="tna-container">
+  <div class="tna-column tna-column--width-3-3 tna-column--full-small tna-column--full-tiny tna-!--margin-top-m">
+    <h2 class="tna-heading-l">Other ways to explore our collection</h2>
+  </div>
+</div>
+  <div class="tna-container">
+    <div class="tna-column tna-column--width-1-3 tna-column--full-small tna-column--full-tiny tna-!--margin-top-m">
+      <article class="tna-card tna-card--demo">
```

Comment:
This could be replaced with the card macro instead which would simplify this code a lot.

Reply:
I've now intergrated the macro card work from James